App.tsx

```diff
@@ -98,6 +100,7 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                 ]}
                 backgroundImage={backgroundImage}
                 searchableConfig={{ placeholder: 'Search', autoFocus: true }}
+                theme={theme.dark ? blueDarkAlt : {}}
```

Comment:
should we make a wrapper for our Header too?

Reply:
We can do that. I just figured this was a one-time use thing and potentially a need to update the component itself.

Reply:
Is the alt theme doing anything other than fixing the blue color on the background? If not, then I would say hardcode the backgroundColor to blue[500] instead of passing in an entire theme and then add a note to your component update story to fix it in the component.